seqr/views/apis/family_api.py

```diff
@@ -41,12 +41,44 @@ def edit_families_handler(request, project_guid):
         return create_json_response(
             {}, status=400, reason="'families' not specified")
 
+    family_guids = [f['familyGuid'] for f in modified_families if f.get('familyGuid')]
+    family_models = {}
+    if family_guids:
+        family_models.update({f.guid: f for f in Family.objects.filter(project=project, guid__in=family_guids)})
+        if len(family_models) != len(family_guids):
+            missing_guids = set(family_guids) - set(family_models.keys())
+            return create_json_response({'error': 'Invalid family guids: {}'.format(', '.join(missing_guids))}, status=400)
+
+        updated_family_ids = {
+            fields[FAMILY_ID_FIELD]: family_models[fields['familyGuid']].family_id for fields in modified_families
+            if fields.get('familyGuid') and fields.get(FAMILY_ID_FIELD) and \
+                fields[FAMILY_ID_FIELD] != family_models[fields['familyGuid']].family_id}
```

Comment:
The `family_models[fields['familyGuid']]` could be undefined when the family with a GUID of `fields['familyGuid']` couldn't be found. Can this case happen?

Reply:
I mean if `family_models[fields['familyGuid']]` is undefined or None, then `family_models[fields['familyGuid']].family_id` will be an error.

Reply:
there is error handling 5 lines above this that returns an error to the user if there isn't a model for any of the guids, so thats completely impossible

Reply:
Okay, I see what you mean.